src/operators/rastertypeconversion.md

```diff
@@ -0,0 +1,42 @@
+# RasterTypeConversion
+
+The `RasterTypeConversion` operator allows change the data type of raster data.
+It transforms all pixels into the new data type.
+
+1. The operator allows loss of precision e.g. converting a `f32` value of "3.1" to a `u8` will return a value of "3".
+
+2. If the old value is not valid in the new type it will clip at the value range of the new type.
```

Comment:
Maybe add an example here as well for non computer scientists :smile:

Reply:
[f61834f](https://github.com/geo-engine/docs/pull/29/commits/f61834f48103b630a6d859ffcffe669c977bceb0)